justfile

```diff
@@ -0,0 +1,74 @@
+name := 'cosmic-comp'
+export APPID := 'com.system76.CosmicComp'
+
+rootdir := ''
+prefix := '/usr'
+
+base-dir := absolute_path(clean(rootdir / prefix))
+
+export SHARE_DIR := sharedir
+export BIN_DIR := base-dir / 'bin'
+export LIB_DIR := libdir
+
```

Comment:
Suggestion: Add a line like `cargo-target-dir := env('CARGO_TARGET_DIR', 'target')` here to read the CARGO_TARGET_DIR environment variable.